packages/messageport/tizen/src/messageport_manager.cc

```diff
@@ -0,0 +1,244 @@
+// Copyright 2021 Samsung Electronics Co., Ltd. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include "messageport_manager.h"
+
+#include <bundle.h>
+#include <flutter/standard_message_codec.h>
+
+#include <cstdint>
+#include <vector>
+
+#include "log.h"
+
+MessagePortManager::MessagePortManager() {}
+
+MessagePortManager::~MessagePortManager() {
+  for (auto& pair : local_ports_) {
```

Comment:
May be simplified to
```suggestion
  for (const auto& [???, port] : local_ports_) {
```